README.asciidoc

```diff
@@ -0,0 +1,31 @@
+= JetBrains IntelliJ IDEA Community
```

Comment:
I know we've been accepting READMEs like this before I don't think it's beneficial at all.

Reply:
I think there is still some value on it. Concise (if they are) configuration/installation instructions save some time. I always try to put the minimum amount of information required for someone to pick it up and get it running in almost no time.

In any case, I have all of that memorized. If you really think it's of no use, I can remove it.

Reply:
We already have quickstart docs and things like application bug tracker or general descriptions belong to appdata.